src/css/info.scss

```diff
@@ -91,6 +91,15 @@
     &-section {
       margin-top: 53px;
 
+      @media (max-width: #{$bp-large}) {
```

Comment:
@astanovferuz , this changes should applied between medium and large size. So, the media should be:
```
@media (min-width: #{$bp-medium}) and (max-width: #{$bp-large - 1}) {
```